extensions/mongodb-client/deployment/src/test/java/io/quarkus/mongodb/MongoTestBase.java

```diff
@@ -1,24 +1,25 @@
 package io.quarkus.mongodb;
 
-import java.io.IOException;
-
 import org.jboss.logging.Logger;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
 
 import de.flapdoodle.embed.mongo.commands.MongodArguments;
 import de.flapdoodle.embed.mongo.config.Net;
 import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.mongo.transitions.ImmutableMongod;
 import de.flapdoodle.embed.mongo.transitions.Mongod;
 import de.flapdoodle.embed.mongo.transitions.RunningMongodProcess;
 import de.flapdoodle.embed.process.types.ProcessConfig;
 import de.flapdoodle.reverse.TransitionWalker;
 import de.flapdoodle.reverse.transitions.Start;
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
What's the reason for this?

Reply:
I guess it's to avoid starting Mongo for every test?